docker-stack-prod.yml

```diff
@@ -45,7 +45,7 @@ services:
             - back_network
         volumes:
             - /var/run/docker.sock:/var/run/docker.sock
-            - /volumes/whitebox_program_uploads:/uploads:ro
```

Comment:
Indeed, it is the 'compilations' directory that needs to be writable. Not the 'uploads' directory. I will add the required lines in the docker-stack-prod.yml file (they were correctly included in the docker-stack-dev.yml file). Good point!